src/main/webapp/app/course/tutorial-groups/tutorial-groups-management/tutorial-groups-checklist/tutorial-groups-checklist.component.html

```diff
@@ -20,18 +20,38 @@ <h1>{{ 'artemisApp.pages.checklist.title' | artemisTranslate }}</h1>
             </a>
         </div>
         <hr />
-        <div class="col-12">
+        <div class="col-12" *ngIf="!isTutorialGroupConfigurationCreated">
             <p>{{ 'artemisApp.pages.checklist.configurationExplanation' | artemisTranslate }}</p>
             <div class="form-check">
-                <input class="form-check-input" (click)="onNoClick($event)" readonly type="checkbox" value="" id="configuration" [checked]="isTutorialGroupConfigurationCreated" />
+                <input class="form-check-input" (click)="onNoClick($event)" readonly type="checkbox" value="" id="configuration" [checked]="false" />
                 <label class="form-check-label" for="configuration">
                     {{ 'artemisApp.pages.checklist.configuration' | artemisTranslate }}
                 </label>
             </div>
-            <a [routerLink]="['/course-management', course.id!, 'create-tutorial-groups-configuration']" class="btn btn-primary btn-md my-2">
+            <a
+                [routerLink]="['/course-management', course.id!, 'create-tutorial-groups-configuration']"
+                class="btn btn-primary btn-md my-2"
+                [class.disabled]="!isTimeZoneConfigured"
+            >
                 <fa-icon [icon]="faPlus"></fa-icon>&nbsp;<span>{{ 'artemisApp.pages.checklist.createConfiguration' | artemisTranslate }}</span>
             </a>
         </div>
+        <div class="col-12" *ngIf="isTutorialGroupConfigurationCreated">
+            <p>{{ 'artemisApp.pages.checklist.configurationExplanation' | artemisTranslate }}</p>
+            <div class="form-check">
+                <input class="form-check-input" (click)="onNoClick($event)" readonly type="checkbox" value="" id="configurationEdit" [checked]="true" />
```

Comment:
What does `onNoClick` do/Why is it there. Could you add a comment above that method?

Reply:
I renamed the method and added an explanation in commit :210a50403f2ee44c78ed1171edd8cf34f9ff7eed. 'readOnly' was not enough to prevent the default click behaviour of a checkbox of 'checking' the checkbox